packages/paste-website/src/pages/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Insights & Metrics that Inform the Paste Design System
+description: Learn how the Twilio Paste design system captures data to gain insights and track metrics which guide the development of the system.
+slug: /articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system/
+date: 2021-04-29
+author: Shadi Isber
+avatar: https://avatars.githubusercontent.com/u/1592327?v=4
+---
+
+import {graphql} from 'gatsby';
+import Img from 'gatsby-image';
+import {Box} from '@twilio-paste/box';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/articles/2021-04-29-insights-&-metrics-that-inform-the-paste-design-system/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+        description
+        date(formatString: "YYYY-MM-DD")
+        author
+        avatar
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    hero: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/insights-metrics-inform-paste-design-system/hero.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    downloads: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/insights-metrics-inform-paste-design-system/downloads.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    search: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/insights-metrics-inform-paste-design-system/search.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+    trends: file(
+      sourceInstanceName: {eq: "assets"}
+      relativePath: {eq: "images/articles/insights-metrics-inform-paste-design-system/trend.png"}
+    ) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+<ArticleHeader
+  title={props.data.mdx.frontmatter.title}
+  description={props.data.mdx.frontmatter.description}
+  date={props.data.mdx.frontmatter.date}
+  author={props.data.mdx.frontmatter.author}
```

Comment:
Would be cool to add the Twitter account in the remark metadata and link it in here behind author names

Reply:
added `author_link`, just incase it's a portfolio site or something